input/drivers_hid/wiiusb_hid.c

```diff
@@ -270,6 +270,44 @@ static int wiiusb_hid_removal_cb(int result, void *usrdata)
    return 0;
 }
 
+static bool isRetrodeGamepad(usb_devdesc devdesc)
+{
+    if (devdesc.idVendor != VID_RETRODE || devdesc.idProduct != PID_RETRODE)
+    {
+        return false;
+    }
+    if (devdesc.configurations != NULL)
+    {
+        if (devdesc.configurations->interfaces != NULL)
+        {
+            if (devdesc.configurations->interfaces->endpoints != NULL)
+            {
+                return devdesc.configurations->interfaces->bInterfaceSubClass == 0;
+            }
+        }
+    }
+    return false;
+}
+
+static bool isRetrodeMouse(usb_devdesc devdesc)
+{
+    if (devdesc.idVendor != VID_RETRODE || devdesc.idProduct != PID_RETRODE)
+    {
+        return false;
+    }
+    if (devdesc.configurations != NULL)
```

Comment:
When a pointer needs to be checked against NULL, instead of 'if (var != NULL)' inside an if statement, just do 'if (var)'.

Reply:
Fixed.